<launch>
  <arg name="nodelet_manager_name" default="standalone_nodelet"/>
  <arg name="start_nodelet_manager" default="true"/>
  <arg name="stargazer_config" default="$(find stargazer_ros_tool)/res/stargazer_optimized.yaml"/>


  <node pkg="stargazer_ros_tool" type="landmark_localizer" name="landmark_localizer_node" output="screen"
        launch-prefix="gdb --args">
    <param name="stargazer_config" value="$(arg stargazer_config)"/>
    <param name="map_frame" value="world"/>
    <param name="robot_frame" value="vehicle"/>
    <param name="use_ceres" value="False"/>
  </node>
</launch>
